app/src/lib/stores/pull-request-store.ts

```diff
@@ -150,7 +158,7 @@ export class PullRequestStore {
       // in the first place.
       const baseRepo = await this.repositoriesStore.findOrPutGitHubRepository(
         repository.endpoint,
-        pr.base.repo!
+        forceUnwrap('PR cannot have a null base repo', pr.base.repo)
```

Comment:
I like this bit of explicitness, but cc @desktop/core for quick feels on this

Reply:
👍